docs/step-by-step/step-03.md

```diff
@@ -13,160 +14,79 @@ and performance](step-01.md).
 
 You might be thinking, "So, now I have to remember all these IP addresses, and type them into my browser
 manually, to move from one system to another? Maybe I should just make a bunch of bookmarks. What's a few more tabs
-on top of the hundred I have already? 🤬"
+on top of the hundred I have already?"
 
-We get it. That's why we built [Netdata Cloud](/docs/netdata-cloud/README.md), which connects many distributed agents
-together for a seamless experience when monitoring multiple systems.
+We get it. That's why we built [Netdata Cloud](https://learn.netdata.cloud/docs/cloud/), which connects many distributed
+agents for a seamless experience when monitoring an entire infrastructure of Agent-monitored nodes.
```

Comment:
Should this be "Netdata-monitored" nodes? i.e. our product rather than one of the components.